tests/testers-rate-limit-wrapper.test.sh

```
@@ -0,0 +1,238 @@
+pass() { printf '  ok %s\n' "$1"; }
+fail() { printf '  FAIL %s: %s\n' "$1" "${2:-}"; FAILS=$((FAILS+1)); }
+
+# Case 1: parse non-numeric --rps-cap=abc
+test_parse_non_numeric() {
+  # Spec §Components: SKILL.md parse-site emits exit 2 with "must be a positive integer" on stderr
+  # Inline harness: extract the parse-validation block from SKILL.md and exercise it
+  out=$(RPS_CAP=abc bash -c '
+    if ! [[ "$RPS_CAP" =~ ^[1-9][0-9]*$ ]]; then
+      echo "error: --rps-cap must be a positive integer (no leading zero, no sign); got '"'"'$RPS_CAP'"'"'" >&2
+      exit 2
+    fi' 2>&1) && { fail "$FUNCNAME" "expected exit 2"; return; }
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)